### PR TITLE
This is the count of characters in a string

### PR DESCRIPTION
**Pull request by Anuva13**

### CODE REVIEW

File: Countchar.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# write a program to count the number of times a letter is appearing in the string
+def main():
+    a = input("enter a string\n")
```
Comment (ight) [inline Countchar.py:3]: please provide proper variable name

Comment (ight) [reply Countchar.py:3]: across the program

File: Countchar.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    b = input("enter the character for which you want to find its number of occurrence in the string\n")
+    c = 0
+    l = len(a)
+    i = 0
+
+    while (i<l):
+
```
Comment (ight) [inline Countchar.py:10]: new line not required

File: Countchar.py (hunk 1 above)
Comment (Anuva13) [reply Countchar.py]: Addressed

File: Countchar.py (hunk 2 above)
Comment (Anuva13) [reply Countchar.py:10]: Addressed

**Review by ight — APPROVED** [review]

**Pull request merged**